features/FEATURE_LWIP/lwip-interface/lwip/src/netif/ppp/lwip_utils.c

```diff
@@ -605,7 +605,11 @@ void ppp_print_string(const u_char *p, int len, void (*printer) (void *, const c
  * ppp_logit - does the hard work for fatal et al.
  */
 static void ppp_logit(int level, const char *fmt, va_list args) {
+#ifdef PPP_LOGIT_BUFSIZE
```

Comment:
So only if you turn on PPP, it sets the PPP_LOGIT_BUFSIZE to 512B. Other wise c code in lwip_utils,h sets it to full 1K as before.